dex/src/instruction.rs

```diff
@@ -741,9 +741,9 @@ pub fn cancel_order(
 ) -> Result<Instruction, DexError> {
     let data = MarketInstruction::CancelOrderV2(CancelOrderInstructionV2 { side, order_id }).pack();
     let accounts: Vec<AccountMeta> = vec![
-        AccountMeta::new_readonly(*market, false),
-        AccountMeta::new_readonly(*market_bids, false),
-        AccountMeta::new_readonly(*market_asks, false),
+        AccountMeta::new(*market, false),
```

Comment:
This is readonly according to the comment https://github.com/project-serum/serum-dex/blob/c4904b9035247027fc72a3b806e62e8d4c1de401/dex/src/instruction.rs#L360.